src/compose/ComposeBox.js

```diff
@@ -445,6 +449,7 @@ const ComposeBox: React$AbstractComponent<Props, ImperativeHandle> = forwardRef(
       } finally {
         setActiveQuoteAndRepliesCount(v => v - 1);
         activeInvocations.current = activeInvocations.current.filter(x => x !== serialNumber);
+        messageInputRef.current?.focus();
```

Comment:
What's the reasoning for putting this step in the `finally` block?

I think it should instead go at the end of the `try` block. The existing job of this `finally` block is purely about making sure that the `setActiveQuoteAndRepliesCount(v => v + 1)` and `activeInvocations.current.push` at the top of the function always get closed eventually, regardless of whether the invocation succeeds or fails. If it does fail, I think it's probably not helpful to go and focus the input — we haven't successfully set up something for the user to type their reply after.

Reply:
Oh interesting, yeah, I agree it seems better to put it at the end of the `try`. Will test to make sure that doesn't break anything.

Reply:
Yep, that seems to work fine. Thanks for the catch!